config/database.php

```diff
@@ -1,30 +1,7 @@
 <?php
 
-// Convert Heroku's `REDIS_URL` to standard `REDIS_HOST`, `REDIS_PORT`,
-// and `REDIS_PASSWORD` environment variables:
-if (env('REDIS_URL')) {
-    $url = parse_url(env('REDIS_URL'));
-
-    putenv('REDIS_HOST='.$url['host']);
-    putenv('REDIS_PORT='.$url['port']);
-    putenv('REDIS_PASSWORD='.$url['pass']);
-}
```

Comment:
[Laravel 5.8 now supports a `REDIS_URL` variable](https://github.com/laravel/laravel/pull/5037/) and will parse it correctly, so no need to do this manually anymore.